### PR TITLE
Add CMake testing on Travis CI.

### PR DESCRIPTION
**Pull request by litghost**

### CODE REVIEW

File: .travis/script.sh [hunk 1]
```diff
@@ -67,3 +67,17 @@ start_section "symbiflow.redir.2" "Running ${GREEN}make${NC} inside ${PURPLE}vpr
...
+
+start_section "symbiflow.configure_cmake" "Configuring CMake"
+mkdir build
+cd build
+cmake ..
+end_section "symbiflow.configure_cmake"
+
+$SPACER
+
+start_section "symbiflow.run_cmake_tests" "Running tests with CMake"
```
Comment (mithro) [inline .travis/script.sh]: I think these might be better as separate sections?

File: .travis/script.sh [hunk 2]
```diff
@@ -67,3 +67,17 @@ start_section "symbiflow.redir.2" "Running ${GREEN}make${NC} inside ${PURPLE}vpr
 end_section "symbiflow.redir.2"
 
 $SPACER
+
```
Comment (mithro) [inline .travis/script.sh]: Should we remove the other make ones?

File: .travis/script.sh (hunk 1 above)
Comment (litghost) [reply .travis/script.sh]: Done

File: .travis/script.sh (hunk 2 above)
Comment (litghost) [reply .travis/script.sh]: Removed

**Pull request merged**